new-docs/pages/v2.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <div>
+    <h1>{{ name }}</h1>
+    <button @click="routeToDocs">Old Docs</button>
+  </div>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      name: 'Hello World'
```

Comment:
Maybe in a follow-up PR, but this should be linted with our normal eslint settings.

Reply:
We have added a followup task to add linting 👍